docs/validators/state-sync.mdx

```diff
@@ -107,4 +107,4 @@ sudo lsof -i :26657
 2. Kill any process running on that port 
 ```bash
 kill -9 PID
```

Comment:
Can this be changed to attempt a graceful exit first? Kill -9 sends the very aggressive SIGKILL which _might_ corrupt the node.
```
pid=$(sudo lsof -t -i :26657) && { timeout 30 sh -c "kill $pid && wait $pid"; [ $? -eq 124 ] && kill -9 $pid; }
```
This command will get the process ID (if any) that is using port 26657, and will send SIGTERM (default for kill) which will instruct the process to exit gracefully. If it isn't complete after 30 seconds, it will send SIGKILL which will force it to close.

If nothing is using port 26657, the behavior will be as follows:
- sudo lsof -t -i :26657 will produce no output, so pid will be set to an empty string.
- The && operator then checks the exit status of the previous command. Since lsof will successfully complete without finding any process (exit status 0), the script on the right side of && will be executed.
- However, since pid is an empty string, the command kill $pid will be effectively kill, which without any arguments does nothing. It won't produce an error but won't have any effect either.
- The timeout command will exit successfully, and the check [ $? -eq 124 ] will fail because the exit status will not be 124 (the exit status for a timeout).
- As a result, kill -9 $pid won't be executed.